apps/web/src/services/cost-explorer/components/BudgetMainListCard.vue

```diff
@@ -120,20 +119,20 @@ const state = reactive({
     >
         <div class="card-header">
             <div class="flex items-center mb-1">
-                <span v-for="(name, index) in state.projects"
-                      :key="name"
-                      class="project-info"
-                      :class="{target: index === state.projects.length - 1}"
+                <span v-for="(targetName, index) in state.targetList"
+                      :key="`${targetName}-${index}`"
+                      class="target-info"
+                      :class="{target: index === state.targetList.length - 1}"
                 >
-                    <p-i v-if="index === state.projects.length - 1"
-                         :name="state.isProject ? 'ic_document-filled' : 'ic_folder-filled'"
+                    <p-i v-if="!storeState.isAdminMode && index === state.targetList.length - 1"
+                         name="ic_document-filled"
                          color="inherit"
                          width="1em"
                          height="1em"
                          class="mr-1"
                     />
-                    {{ name }}
-                    <p-i v-if="index < state.projects.length - 1"
+                    {{ targetName }}
+                    <p-i v-if="index < state.targetList.length - 1"
                          name="ic_chevron-right-thin"
                          width="1em"
                          height="1em"
```

Comment:
description:

In admin mode
![스크린샷 2023-12-12 오후 10 31 38](https://github.com/cloudforet-io/console/assets/18563857/49ed884c-c366-4ebb-a2ab-d38fd7b70b2e)

In user mode
![스크린샷 2023-12-12 오후 10 31 47](https://github.com/cloudforet-io/console/assets/18563857/223a2a3f-a25d-42a1-8dc4-6428c3db98d0)